test_fms/fms2_io/test_io_simple.F90

```diff
@@ -0,0 +1,90 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+! This is a very simple test of FMS IO.
+
+! Ed Hartnett, 6/10/20
```

Comment:
These comments should be in doxygen style

Reply:
It would be great if I knew more about the use of domains, but this code is copied from test_atmosphere_io.F90, and existing test which is currently being skipped. So perhaps you could tell me about your domains, or direct me to some documentation that explains them. ;-)

Reply:
This edit request has nothing to do with domains.  This code needs to be changed to 
```fortran
!> \brief This is a very simple test of FMS IO.
!> \author Ed Hartnett, 6/10/20
```
The comment style needs be in doxygen.

Reply:
OK, fixed.